tests/test_observatory.py

```diff
@@ -217,3 +220,38 @@ def test_observatories_registered():
 
 def test_gbt_registered():
     get_observatory("gbt")
+
+
+def test_is_gbt_still_ok():
+
+    gbt = get_observatory("gbt")
+    assert gbt._loc_itrf.y < 0
+
+
+@pytest.mark.parametrize("overwrite", [True, False])
+def test_observatory_override(overwrite):
```

Comment:
This still appears to trash the GBT observatory coordinates. Perhaps a fixture that saves and restores the registry?

Reply:
I think this still changes the GBT position

Reply:
Yes, you may be right (I hadn't realized that the environment persisted across different tests). For the moment I just added:
`    load_observatories_from_usual_locations(clear=True)` but I guess I can do a fixture if needed.